ppx/common.ml

```diff
@@ -11,7 +11,8 @@ let core_types loc =
     ; ("int64", [%expr Rpc.Types.(Basic Int64)])
     ; ("string", [%expr Rpc.Types.(Basic String)])
     ; ("float", [%expr Rpc.Types.(Basic Float)])
-    ; ("bool", [%expr Rpc.Types.(Basic Bool)]) ]
+    ; ("bool", [%expr Rpc.Types.(Basic Bool)])
+    ; ("base64", [%expr Rpc.Types.(Basic Base64)]) ]
```

Comment:
Please revert this change. 
At this stage, base64 is not a core type. Like DateTime it needs to be handled by hand.

Reply:
OK done.